src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">
+            Surname:
+            <input
+              class="form-field"
+              type="text"
+              name="surname"
+              id="surname"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="age">
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              id="age"
+              value="12"
+              min="1"
+              max="100"
+            >
+          </label>
+        </div>
+        <div class="form-field">
+          <label for="dateOfBirth">
+            Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+              id="dateOfBirth"
+            >
+          </label>
+        </div>
+        <div>
+          <label for="terms">
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+              id="terms"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="groups">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+        <div>
+          <label for="password">
+            Password:
+            <input
+              type="password"
+              name="password"
+              id="password"
+              minlength="5"
+              maxlength="15"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="groups">
```

Comment:
it's clearer to name like this
```suggestion
      <fieldset class="fieldset">
```